cypress/e2e/reviewApplication.feature

```diff
@@ -1,7 +1,9 @@
+@processes
 @authentication
 @common
-@processes
 @root
+@personal-details
+@worktray
```

Comment:
No idea what this feature is, but I suggest reviewing the need to run this spec when these MFEs receive changes. Is this feature and its tests really that related to all these listed MFEs?